examples/todomvc/tests/integration.spec.ts

```diff
@@ -2,7 +2,6 @@
 
 import { test, expect } from '@playwright/test';
 import type { Page } from '@playwright/test';
-
```

Comment:
Why this stray change? Let's revert it.

Reply:
That's right. I think I committed by mistake. I'll correct it.